### PR TITLE
Minor map changes

### PR DESCRIPTION
**Pull request by Dzierzan**
Tomb raiders has now tech drill rigs.
Added more stuff for shellmap.

### CODE REVIEW

File: mods/hv/maps/tomb-raiders/map.yaml [hunk 1]
```diff
@@ -6,8 +6,6 @@ Title: Tomb Raiders
 
 Author: Graion Dilach
 
-License: CC-BY-SA-4.0
```
Comment (Mailaender) [inline mods/hv/maps/tomb-raiders/map.yaml:9]: Please don't remove that.

File: mods/hv/maps/tomb-raiders/map.yaml [hunk 2]
```diff
@@ -219,17 +219,17 @@ Actors:
...
 		Owner: Neutral
-		Location: 47,53
-	Actor80: techbuilding
+		Location: 47,52
+	Actor80: techminer
 		Owner: Neutral
-		Location: 47,31
-	Actor81: techbuilding
+		Location: 47,30
+	Actor81: techminer
```
Comment (Mailaender) [inline mods/hv/maps/tomb-raiders/map.yaml:231]: I think @GraionDilach's whole point of the map was that the tech reconstructors stack up and give more healing.

File: mods/hv/maps/tomb-raiders/map.yaml (hunk 1 above)
Comment (Dzierzan) [reply mods/hv/maps/tomb-raiders/map.yaml:9]: Goddamn, edit the engine, so it's automatically added or something.

File: mods/hv/maps/tomb-raiders/map.yaml (hunk 2 above)
Comment (Dzierzan) [reply mods/hv/maps/tomb-raiders/map.yaml:233]: @GraionDilach doesn't mind.

Comment (GraionDilach) [reply mods/hv/maps/tomb-raiders/map.yaml:233]: At that point we had no other tech building. I don't mind this change. When limited eco comes in, we need to revisit the eco on all the amps anyway.

**Pull request merged**